### PR TITLE
document code of conduct

### PR DESCRIPTION
**Pull request by stapelberg**
fixes https://github.com/i3/i3/issues/2676

Largely copied from django (with attribution). Changed contact addresses and removed paragraphs relating to django events or their organizational structure.

### CODE REVIEW

File: conduct-faq.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+<h2>Code of Conduct: FAQ</h2>
+
+<p>
+Largely taken from <a href="https://www.djangoproject.com/conduct/faq/">django</a>. Thanks!
+</p>
+
+<p>
+This FAQ attempts to address common questions and concerns around the i3
+community's <a href="/conduct">Code of Conduct</a>. If you still have
+questions after reading it, please feel free to contact us at conduct AT i3wm DOT org.
```
Comment (Airblader) [inline conduct-faq.html]: So I assume this email address actually exists? :-)

File: conduct.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+<div id="content">
+
+<h2>Code of Conduct</h2>
+
+<p>
+Largely taken from <a href="https://www.djangoproject.com/conduct/">django</a>. Thanks!
+</p>
+
+<p>Like the technical community as a whole, the Django team and community is made up
```
Comment (Airblader) [inline conduct.html]: s/Django/i3

File: conduct.html [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+<p>Diversity is one of our huge strengths, but it can also lead to communication
+issues and unhappiness. To that end, we have a few ground rules that we ask
+people to adhere to. This code applies equally to founders, mentors and those
+seeking help and guidance.</p>
+
+<p>This isn’t an exhaustive list of things that you can’t do. Rather, take it in
+the spirit in which it’s intended - a guide to make it easier to enrich all of
+us and the technical communities in which we participate.</p>
+
+<p>This code of conduct applies to all spaces managed by the Django project or
```
Comment (Airblader) [inline conduct.html]: Lots of Django in here

File: conduct.html [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+  decisions. Remember that we're a world-wide community, so you might not be
+  communicating in someone else's primary language.</li>
+
+  <li><strong>Be respectful.</strong> Not all of us will agree all the time, but disagreement is no
+  excuse for poor behavior and poor manners. We might all experience some
+  frustration now and then, but we cannot allow that frustration to turn into a
+  personal attack. It’s important to remember that a community where people feel
+  uncomfortable or threatened is not a productive one. Members of the Django
+  community should be respectful when dealing with other members as well as with
+  people outside the Django community.</li>
```
Comment (Airblader) [inline conduct.html]: Djangooooooo

File: conduct.html [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+    <li>Posting (or threatening to post) other people's personally identifying information ("doxing").</li>
+    <li>Personal insults, especially those using racist or sexist terms.</li>
+    <li>Unwelcome sexual attention.</li>
+    <li>Advocating for, or encouraging, any of the above behavior.</li>
+    <li>Repeated harassment of others. In general, if someone asks you to stop, then stop.</li>
+  </ul>
+  </li>
+
+  <li><strong>When we disagree, try to understand why.</strong> Disagreements, both social and
+  technical, happen all the time and Django is no exception. It is important that
```
Comment (Airblader) [inline conduct.html]: Django No. 5

File: conduct-faq.html (hunk 1 above)
Comment (stapelberg) [inline conduct-faq.html]: It does. You’re now also added as an owner.

**Comment by stapelberg** [comment]
All the occurrences of django are now replaced. You can clearly tell that I only started doing that systematically when I got to conduct-{faq,reporting}.html, but now conduct.html should also be good :).

File: conduct-faq.html (hunk 1 above)
Comment (Airblader) [inline conduct-faq.html]: Then I guess I should learn how to access it. :-) Perhaps I should also have my own email address under that domain for releases?

Comment (stapelberg) [inline conduct-faq.html]: You’ll automatically get emails sent to that address (it’s a group).

If you want, I can set up an i3wm.org email account for you, but note that it’s not a pre-requisite for releases — I never used an i3wm.org address for releases. In any case, let’s continue this discussion outside of this pull request :).

**Pull request merged**